spec/index.bs

```diff
@@ -8075,6 +8077,14 @@ dictionary GPUFragmentState
                     : {{GPUTextureSampleType/"uint"}}
                     :: |output| must have an unsigned integer scalar type.
                 </dl>
```

Comment:
Do the two rules above this apply to src1? (Does src1 actually need to match the format, etc.?)